components/EmojiPopover/EmojiPopover.module.scss

```diff
@@ -26,16 +32,19 @@
 .emojiItem {
   font-size: 1.2rem;
   cursor: pointer;
-  padding: 2px 2px 2px 4px;
-
+  display: flex;
```

Comment:
<img width="435" alt="Screenshot 2023-08-02 at 16 38 08" src="https://github.com/Juratbek/upper_frontend/assets/55176773/fc778e64-7c5b-40ff-9661-67aebfd5d57e">

emoji container should be in same width with the footer